validator/src/main/resources/expected-data-template/python/eks/rds-mysql-trace.mustache

```diff
@@ -0,0 +1,94 @@
+[
+  {
+    "name": "^{{serviceName}}$",
+    "origin": "^AWS::EKS::Container$",
+    "subsegments": [
+      {
+        "name": "^mysql$",
+        "namespace": "^remote$",
+        "aws": {
+          "account_id": "^{{accountId}}$",
+          "xray": {
+            "auto_instrumentation": true,
+            "sdk": "^opentelemetry for python$"
+          }
+        },
+        "sql": {
+          "connection_string": "^localhost/information_schema$",
+          "url": "^SELECT$",
+          "sanitized_query": "^SELECT \\* FROM tables LIMIT 1;$",
+          "database_type": "^mysql$"
+        },
+        "metadata": {
+          "default": {
+            "aws.span.kind": "^CLIENT$",
+            "EKS.Cluster": "^e2e-python-canary-test$",
+            "PlatformType": "^AWS::EKS$"
+          }
+        },
+        "annotations": {
+          "aws.local.service": "^{{serviceName}}$",
+          "aws.local.operation": "^GET mysql$",
+          "aws.remote.resource.type": "^DB::Connection$",
+          "aws.remote.resource.identifier": "^{{remoteResourceIdentifier}}$",
+          "aws.remote.service": "^mysql$",
+          "aws.remote.operation": "^SELECT$"
+        }
+      }
+    ],
+    "http": {
+      "request": {
+        "url": "^{{endpoint}}/mysql$",
+        "method": "^GET$"
+      },
+      "response": {
+        "status": 200
```

Comment:
Need assurances that this is an appropriate expectation. The traces we generate could pretty easily fail on one of the attempts to ping the sample app, and if we pull that trace then the test will fail. Some amount of randomness likely exists here.

I understand that our existing test cases already expect `200` response code, but let's make sure expecting that here is also reasonable.

Reply:
Sure